staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -134,6 +128,7 @@ func NewPrefixTransformers(err error, transformers ...PrefixTransformer) Transfo
 // the result of transforming the value. It will always mark any transformation as stale that is not using
 // the first transformer.
 func (t *prefixTransformers) TransformFromStorage(data []byte, context Context) ([]byte, bool, error) {
+	start := time.Now()
```

Comment:
Should not this be moved into the loop?
Otherwise the start time for all transformations will be the same.

Reply:
line#146 requires start time being set before for loop. it also makes sense because only one of the transformers will be called.

Reply:
You are correct.

Reply:
/assign @liggitt